resume.json

```diff
@@ -1,108 +1,196 @@
 {
   "basics": {
     "name": "Nuvi Pannu",
-    "label": "Programmer",
-    "picture": "",
-    "email": "npannu@gmail.com",
-    "phone": "(912) 555-4321",
-    "website": "http://nuvipannu.com",
-    "summary": "<3",
+    "label": "Developer/Tech ♡er",
+    "picture": "http://i.imgur.com/ns7polk.jpg?1",
```

Comment:
@nuvipannu I cropped the profile picture to be a square. Before, it was making you fat because of the picture proportions. :joy: